exporter/redis.go

```diff
@@ -624,11 +624,7 @@ func (e *Exporter) extractInfoMetrics(info, addr string, alias string, scrapes c
 			continue
 		}
 
-		split := strings.Split(line, ":")
-		if len(split) != 2 {
```

Comment:
I'd keep the `len == 2` check. What if there's no `:` in the string?

Reply:
Isn't that the check before it?

https://github.com/oliver006/redis_exporter/blob/e3f43cfe305f8d20d931de6eb84e32c664b7edd1/exporter/redis.go#L623-L625

Reply:
You're very right, didn't see that in the diff. Good to go!